packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -100,6 +100,7 @@ const StyledList = styled.div`
 `;
 
 const StyledInnerList = styled.div`
+  max-height: calc(100vh - ${({ theme }) => theme.spacing(19)});
```

Comment:
FYI for whoever reviews this PR ; https://github.com/twentyhq/twenty/pull/6209#issuecomment-2222115237

Reply:
Thanks for pointing that out, @FelixMalfait. I switched to using dvh and also adjusted the max-height for mobile screens, which I had missed in the initial commit. I appreciate the feedback—please let me know if this addresses your concern.